code/modules/projectiles/ammo_types/xenos/pyrogen_xenoammo.dm

```diff
@@ -27,3 +27,5 @@
 		new /obj/fire/melting_fire(affecting)
 		for(var/mob/living/carbon/fired in affecting)
 			fired.take_overall_damage(PYROGEN_FIREBALL_AOE_DAMAGE, BURN, FIRE, FALSE, FALSE, TRUE, 0, , max_limbs = 2)
+		for(var/obj/hitbox/fired in affecting)
```

Comment:
don't loop twice.

Reply:
Do you want me to combine the two loops into 1 single loop, or add a continue to the end of each one?

Reply:
do a single loop, you're just making twice as much work.

Reply:
Alright, done